cql/ScreeningAverageRiskLibrary.cql

```diff
@@ -391,14 +394,21 @@ define RecommendationText:
             }
         else if Age21Through29 then
           { 'Cervical cancer screening using cervical cytology testing alone is next due on ' + ToString(ToDate(ProposedScreeningDate)) + '.' }
-        else
+        else if WillBeAge65OrLessAtNextScreening then
             {
               'Cervical cancer screening is next due on ' + ToString(ToDate(ProposedScreeningDate)) + '.',
               'The screening can be performed using one of the following screening approaches:',
               '1. hrHPV testing alone',
               '2. Cotesting with cervical cytology and hrHPV testing',
               '3. Cervical cytology testing alone'
             }
+        else 
+            {
+              'No cervical cancer screening is recommended.',
+              'Per USPSTF guidelines, this patient will be older than 65 by the time of their next screening, and has been adequately screened in the past. Therefore, they no longer require cervical cancer screening.',
+              'Adequate prior screening is defined as having either three consecutive negative cytology results, or two consecutive negative hrHPV or cotest results, within the past 10 years, with the most recent test occurring within the past 5 years, before stopping screening.'
+            }
+
   else
     null
 
```

Comment:
Should we be displaying dates when no cervical cancer screening is recommended? I also think the short text in this file should be updated to be more dynamic than just displaying 'Cervical Screening Due Now' or 'Cervical Screening Up To Date' - we may want to display when a patient can stop screening.

Reply:
I added [CCSMCDS-337](https://jira.mitre.org/browse/CCSMCDS-337) to address this, since these comments are beyond the scope of this PR.